sklearn/linear_model/ridge.py

```diff
@@ -624,7 +624,10 @@ class Ridge(_BaseRidge, RegressorMixin):
 
     See also
     --------
-    RidgeClassifier, RidgeCV, :class:`sklearn.kernel_ridge.KernelRidge`
+    RidgeClassifier : Ridge classifier
+    RidgeCV : Ridge regression with built-in cross validation
+    :class:`sklearn.kernel_ridge.KernelRidge` : Kernel ridge regression
```

Comment:
Why do you use such syntax here?
What about just `kernel_ridge.KernelRidge`?

Reply:
Note it was already like that before and the link works properly (I checked at some point I think). I am not sure but maybe this is to ensure there is a proper link in the doc. Wild guess: maybe the link is found automatically if the name is in the same module, but maybe not if it's another subpackage.